src/cow-react/modules/wallet/web3-react/hooks/useGnosisSafeSdkInfo.ts

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from 'react'
+
+import { GnosisSafe } from '@web3-react/gnosis-safe'
+import { Connector } from '@web3-react/types'
+import type { SafeInfo } from '@gnosis.pm/safe-apps-sdk'
+
+export type GnosisSafeSdkInfo = SafeInfo
+
+export function useGnosisSafeSdkInfo(connector: Connector, active?: boolean): GnosisSafeSdkInfo | null {
```

Comment:
This hook is similar to the one called before `useGnosisSafeInfo`. 

I added the word `Sdk` to make sure we understand this hook only applies to that provider/connector.


This hook is not exposed out of the `wallet` module any more. Now is used just as a helper hook for the updater